docs/excel/excel-add-ins-core-concepts.md

```diff
@@ -157,18 +178,16 @@ range.format.fill.color =  null;
 
 Formatting properties such as `size` and `color` will contain `null` values in the response when different values exist in the specified range. For example, if you retrieve a range and load its `format.font.color` property:
 
-- If all cells in the range have the same font color, `range.format.font.color` specifies that color.
-- If multiple font colors are present within the range, `range.format.font.color` is `null`.
+* If all cells in the range have the same font color, `range.format.font.color` specifies that color.
```

Comment:
Just curious about the change from `-` to `*` since the output should be the same...?

Reply:
The docs.microsoft.com VS Code extension I have flags `-` as a warning. I don't really know why, but who am I to argue with a yellow line?

Reply:
I don't even know what to say to that because my extension recommends a dash instead of an asterisk...lol.
![image](https://user-images.githubusercontent.com/28541963/88859433-94d93900-d1ae-11ea-8f2e-68ef726c9d8b.png)

Reply:
Great... Well, they render correctly and that's the important thing. I'll stop changing bulleted lists in existing articles.

Reply:
The [guidance in the contributor guide](https://docs.microsoft.com/en-us/contribute/markdown-reference#bulleted-list) is to use the same notation consistently in an article. Seems like there's no preference between `-` and `*`.